econml/inference/_inference.py

```diff
@@ -1242,7 +1242,7 @@ def pvalue(self, *, value=0):
         pvalue = norm.sf(np.abs(self.zstat(value=value)), loc=0, scale=1) * 2
         return pvalue
 
-    def conf_int_mean(self, *, alpha=.1):
+    def conf_int_mean(self, *, alpha=None):
```

Comment:
Could you also please add a test to test_inference.py that fails with the current implementation but will be fixed by your changes?  For example, I believe that a test that trains an estimator, calls `ate_interval` with two different values of alpha, and asserts that the interval from the smaller alpha is wider would do this.

Reply:
Should be done with the recent commits